### ISSUE TITLE
Add support for sampled netflow v9

### ISSUE THREAD

**Issue by pavel-odintsov** [pavel-odintsov/fastnetmon#141]
Unfortunately, sampled netflow did not handled at all. More details here: http://en.wikipedia.org/wiki/NetFlow#Sampled_NetFlow

**Comment by pavel-odintsov** [comment]
Some work in https://github.com/FastVPSEestiOu/fastnetmon/commit/40d71680d9f12a08de600f1850731f40166a63c5

**Comment by pavel-odintsov** [comment]
This task is really challenging http://sourceforge.net/p/nfsen/mailman/message/30309966/ :( We should parse OPTIONS fields.

**Comment by pavel-odintsov** [comment]
And we haven't this code in flowd:
```C
       case NF9_OPTIONS_FLOWSET_ID:
                        / XXX: implement this (maybe) /
                        logit(LOG_DEBUG, "netflow v.9 options flowset");
                        break;
```

So sad :(

**Comment by p-sherratt** [comment]
looks like a pain to implement.. but will support for this likely be added any time soon?

**Comment by pavel-odintsov** [comment]
Yep, it's definitely pain to implement.... Are you really need this? Why?

Could you try work around:
```bash
# Netflow v9 and IPFIX agents use different and very complex approaches for notifying about sample ratio
# Here you could specify a sampling ratio for all this agents
# For NetFLOW v5 we extract sampling ratio from packets directely and this option not used
netflow_sampling_ratio = 1
```

**Comment by p-sherratt** [comment]
we have a mix of sampling ratios depending on the border device... we could get away with manually defining per device, but this isn't an option in the config either, and I'd rather not have to drop down sampling rate everywhere to accommodate a fixed ratio :(

**Comment by pavel-odintsov** [comment]
Yes, you have only this option now :( We are already reworked sflow paraser and netflow will be down the way.

**Comment by pavel-odintsov** [comment]
Unfortunately, for Junos and MX240 / MX960 / MX480 we haven't any information about sampling rate inside of netflow.

More information:
https://sourceforge.net/p/nfsen/mailman/message/32507759/
http://permalink.gmane.org/gmane.network.nsp.juniper/18922

**Comment by pavel-odintsov** [comment]
<img width="786" alt="netflow_datagram" src="https://cloud.githubusercontent.com/assets/2744166/15509782/69c3074e-21dd-11e6-866a-ffee822df1c8.png">

**Comment by pavel-odintsov** [comment]
Here you could find options template.

<img width="710" alt="flow_template" src="https://cloud.githubusercontent.com/assets/2744166/15509830/95b5104a-21dd-11e6-9522-a55ba562e8ed.png">

**Comment by pavel-odintsov** [comment]
As you can see we haven't any information about actual sampling rate (1:100).

**Comment by pavel-odintsov** [comment]
This bug was fixed in one of recent Junos releases.

**Comment by pavel-odintsov** [comment]
We implemented support for sampled Netflow v9 in FastNetMon Advanced: https://fastnetmon.com/fastnetmon-advanced/ 

it still need a lot of work for community version to implement this option.

**Comment by pavel-odintsov** [comment]
I've merged new Netflow logic for 1.1.7 release which includes sampling support for Netflow v9: https://github.com/pavel-odintsov/fastnetmon/pull/851
